src/engine/SCons/Variables/__init__.py

```diff
@@ -287,9 +288,13 @@ def GenerateHelpText(self, env, sort=None):
 
         env - an environment that is used to get the current values
               of the options.
+        cmp - Either a function as follows: The specific sort function should take two arguments and return -1, 0 or 1 
+              or a boolean to indicate if it should be sorted.
```

Comment:
this might read less awkwardly if reversed: Either True/False to indicate whether to sort using defaults, or the name of a function to use for sorting. The specific sort function should take two arguments and return -1, 0 or 1.